index.js

```diff
@@ -8,6 +8,8 @@ var mkdirp     = require('mkdirp')
 var rimraf     = require('rimraf')
 var mdm        = require('mdmanifest')
 var fs         = require('fs')
+var valid      = require('muxrpc-validation')
+require('./validators')
```

Comment:
Why are you requiring ./validators without using a return value or passing it anything? seeing a line like this makes me think that it adds globals, but it doesn't seem to do this, so it must mutate some other shared state? It's function would be way more obvious if it operated on something that was passed to it. (or did not alter state, but just returned a value)